__tests__/services/UriBuilder.spec.ts

```diff
@@ -0,0 +1,187 @@
+import { MethodKind } from '../../src/models/kinds/MethodKind';
+import { MethodOperation } from '../../src/models/kinds/MethodOperation';
+import { IMethodModel } from '../../src/models/MethodModel';
+import { lowerFirst } from '../../src/utils';
+import { UriBuilder } from '../../src/services/UriBuilder';
+import { ParameterPlace } from '../../src/models/kinds/ParameterPlace';
+
+describe('UriBuilder tests', () => {
+    let uriBuilder: UriBuilder;
+    beforeEach(() => (uriBuilder = new UriBuilder()));
+
+    describe('buildUri', () => {
+        describe('return correct name', () => {
+            test('method with one path param', () => {
+                // Arrange
+                const model: IMethodModel = {
+                    kind: MethodKind.Default,
+                    name: lowerFirst('get'),
+                    operation: MethodOperation.GET,
+                    parameters: [
+                        {
+                            dtoType: "string",
```

Comment:
use single quotes instead of double
as set in prettier

Reply:
Done